udtraek.py

```diff
@@ -100,6 +100,29 @@ def settlement(self, start_date='2000-01-01', end_date='2019-01-01'):
         return pd.DataFrame(results)
 
 
+    def multi_wdex(self, ncpus=None):
+        if ncpus is None:
+            ncpus = cpu_count() -1
+        cols = ['NYMFID', 'TRANSACTION_DATE', 'PARENT_ID', 'AMOUNT', 'FT_TYPE_FLG']
+        temp = self.df.loc[:, cols]
+        nymfids = temp['NYMFID'].unique()
+        temp['AMOUNT'] = to_amount(temp['AMOUNT'])
+        pool = ProcessingPool(ncpus=ncpus)
+        result = pool.map(lambda x: _multi_wdex_worker(temp, x), nymfids)
+        return list(filter(None,result))
+
+def _multi_wdex_worker(df, nymfid):
+    d = df.loc[df['NYMFID'] == nymfid]
+    d = d.loc[d['PARENT_ID'].str[-4:] == 'WDEX']
+    if len(d) >= 2:
+        for date, g in d.groupby(d.TRANSACTION_DATE.dt.date):
+            if len(g) >= 2:
+                if len(g.loc[g['FT_TYPE_FLG'] == 'AD']) == len(g.loc[g['FT_TYPE_FLG'] == 'AX']):
```

Comment:
Lad os hive `len(g.loc[g['FT_TYPE_FLG'] == 'AD'])` og `len(g.loc[g['FT_TYPE_FLG'] == 'AX'])` ud i to variable så det bliver lidt nemmere at læse. De bruges også nedenfor.